src/parse/matches/arg_matches.rs

```diff
@@ -378,7 +382,7 @@ impl ArgMatches {
     pub fn value_of_t_or_exit<R>(&self, name: &str) -> R
     where
         R: FromStr,
-        <R as FromStr>::Err: Display,
+        <R as FromStr>::Err: Display + Debug,
```

Comment:
Another debug

Reply:
Because of forwarding it to `value_of_t`